packages/services/f-http/README.md

```diff
@@ -151,31 +118,26 @@ const wrapper = mount(MyComponent, {
 });
 ```
 
-### Integration Testing Guidance
-F-Http exposes a way to create a mockClient, this enables you to mock the underlying HTTPProvider with responses.
+### Integration / System Testing Guidance
+The module exposes a way to create a MockClient, so you can mock the underlying API with pre-configured responses.
 
 ```js
-import httpModule from '@justeat/f-http';
+import { httpVerbs, MockFactory, CreateClient } from '@justeat/f-http';
```

Comment:
I don't think we have a recommended way at the moment; mostly trying both out until one seems like its better than another, it works either way.

I think I am leaning towards the destructured approach simply because it reduces the code footprint by a tiny amount, but I haven't settled yet. Ill probably revisit the way we document that once we have completed an MVP as the package isn't production ready yet.